src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -294,19 +292,16 @@ function deleteQuestion(number) {
  * Allows users to cancel editing questions
  */
 function cancelEdit(number) {
-    if (number === -1) {
-        if (confirm('Are you sure you want to cancel adding this question?')) {
-            location.reload();
-        }
-    } else {
-        if (confirm('Are you sure you want to cancel your changes?')) {
-            location.reload();
-        }
+    var confirmationMsg = number === -1 ?
+            'Are you sure you want to cancel adding this question?' :
+            'Are you sure you want to cancel your changes?';
```

Comment:
That'll be nice - rather than having two conflicting guides, we can make one that is unified for Teammates development. I'll start exploring and see if which is most consistent with our Java style guide and let you know again.